src/formatter.cc

```diff
@@ -370,9 +370,12 @@ int Formatter::ParseDurationString(const std::string value) {
 
     // 15
     if (input.find(".") == std::string::npos) {
-        int minutes = 0;
-        if (Poco::NumberParser::tryParse(input, minutes)) {
-            return seconds + (minutes * 60);
+        Poco::Int64 minutes = 0;
+        if (Poco::NumberParser::tryParse64(input, minutes)) {
+            if (minutes > kMaxDurationSeconds) {
+                return kMaxDurationSeconds;
```

Comment:
I'm suppose that If it's incorrect when we return `kMaxDurationSeconds` in `Formatter::ParseDurationString`.

Let take a following code as a typical example:

https://github.com/toggl/toggldesktop/blob/91d8d9fa02ea56ddb3fc42527a8584f2a31e808f/src/context.cc#L2790-L2793

Let imagine that in some scenario, the`Formatter::ParseDurationString` return `kMaxDurationSeconds`, but the 
```
return displayError(error(kOverMaxDurationError));
```
is never executed.

---------
I could be fixed by replacing with
```
if ((minutes * 60) > kMaxDurationSeconds) {
        return kMaxDurationSeconds + 1;
}
```

Reply:
I think the `kMaxDurationSeconds + 1` is not needed we can just use
```
if (seconds >= kMaxDurationSeconds) { 
     return displayError(error(kOverMaxDurationError)); 
 }
```
In `context.cc` and it will work. Fixed with (43093445d8947dff89c22e8e44c41685894d1b91)